docs/rst/tutorials/cloud/router_conf_with_manual_topics/router_conf_with_manual_topics.rst

```diff
@@ -0,0 +1,202 @@
+------------------
+
+Let us start by running the ROS 2 *talker* and *listener* nodes.
+
+Environment setup
+^^^^^^^^^^^^^^^^^
+
+To run the nodes, we need to set up the Vulcanexus environment so that the ``demo_nodes_cpp`` package is available.
+There are two ways to achieve this:
+
+#.  Running the Vulcanexus Docker image.
+
+    Run the Vulcanexus Docker image by executing:
+
+    .. code-block:: bash
+
+        docker run -it ubuntu-vulcanexus:iron-desktop
+
+    And then, source the Vulcanexus installation by executing (inside the container):
+
+    .. code-block:: bash
+
+            source /opt/vulcanexus/iron/setup.bash
+
+#.  Setting up a development environment on the local host.
+
+    To do this, the ``vucanexus-iron-desktop`` package is needed, since it includes all the simulation tools, demos, and tutorials.
+
+    Set up the Vulcanexus environment by executing:
+
+    .. code-block:: bash
+
+            source /opt/vulcanexus/iron/setup.bash
+
+Running ROS 2 nodes
+^^^^^^^^^^^^^^^^^^^
+
+Publish in domain 0 and subscribe in domain 1
+"""""""""""""""""""""""""""""""""""""""""""""
+
+Run a ROS 2 ``demo_nodes_cpp`` *talker* on domain ``0``:
+
+.. code-block:: bash
+
+    ROS_DOMAIN_ID=0 ros2 run demo_nodes_cpp talker
+
+Run a ROS 2 ``demo_nodes_cpp`` *listener* on domain ``1``:
+
+.. code-block:: bash
+
+    ROS_DOMAIN_ID=1 ros2 run demo_nodes_cpp listener
+
+Since the |rosrouter| does not have a manual topic to configure the ``max-tx-rate`` for the topic ``secret`` on the ``ROS_2_Domain_1`` participant, the listener will receive messages at an unlimited rate and it will print them in ``stdout``.
+
+
+Publish in domain 0 and subscribe in domain 1 on topic secret
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Run a ROS 2 ``demo_nodes_cpp`` *talker* on domain ``0``:
+
+.. code-block:: bash
+
+    ROS_DOMAIN_ID=0 ros2 run demo_nodes_cpp talker --ros-args -r chatter:=secret
+
+Run a ROS 2 ``demo_nodes_cpp`` *listener* on domain ``1``:
+
+.. code-block:: bash
+
+    ROS_DOMAIN_ID=1 ros2 run demo_nodes_cpp listener --ros-args -r chatter:=secret
+
+Since the |rosrouter| has a manual topic to configure the ``max-tx-rate`` for the topic ``secret`` on the ``ROS_2_Domain_1`` participant, the listener will receive messages at most at 1Hz and it will print them in ``stdout``.
+
+
+Publish in domain 1 and subscribe in domain 0 on topic secret
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Run a ROS 2 ``demo_nodes_cpp`` *talker* on domain ``1``:
+
+.. code-block:: bash
+
+    ROS_DOMAIN_ID=1 ros2 run demo_nodes_cpp talker
+
+Run a ROS 2 ``demo_nodes_cpp`` *listener* on domain ``0``:
+
+.. code-block:: bash
+
+    ROS_DOMAIN_ID=0 ros2 run demo_nodes_cpp listener
+
+Since the |rosrouter| does not have a manual topic to configure the ``max-tx-rate`` for the topic ``secret`` on the ``ROS_2_Domain_0`` participant, the listener will receive messages at an unlimited rate and it will print them in ``stdout``.
```

Comment:
```suggestion
Publish in domain 1 and subscribe in domain 0 on topic secret
"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""

Run a ROS 2 ``demo_nodes_cpp`` *talker* on domain ``1``:

.. code-block:: bash

    ROS_DOMAIN_ID=1 ros2 run demo_nodes_cpp talker --ros-args -r chatter:=secret

Run a ROS 2 ``demo_nodes_cpp`` *listener* on domain ``0``:

.. code-block:: bash

    ROS_DOMAIN_ID=0 ros2 run demo_nodes_cpp listener --ros-args -r chatter:=secret

Since the |rosrouter| does not have a manual topic to configure the ``max-tx-rate`` for the topic ``secret`` on the ``ROS_2_Domain_0`` participant, the listener will receive messages at an unlimited rate and it will print them in ``stdout``.
```